web/src/stream_create.js

```diff
@@ -360,7 +360,34 @@ export function show_new_stream_modal() {
             });
         }
     }
+    const $add_subscribers_container = $(
+        "#stream_creation_form .add-subscribers-container",
+    ).expectOne();
 
+    const $input_element = $add_subscribers_container.find(".input").expectOne();
+    const $add_subscribers_button = $add_subscribers_container.find("button[name=add_subscriber]");
+
+    const $subscribe_all_users_button = $add_subscribers_container.find(
+        "button.add_all_users_to_stream",
+    );
+
+    if (settings_data.user_can_subscribe_other_users()) {
+        $add_subscribers_container.removeClass("add_subscribers_disabled_tooltip");
+
+        $input_element.prop("contenteditable", true);
```

Comment:
Some part of this is duplicated from the code in `stream_ui_updates.update_add_subscriptions_elements`. but I don't think it would help much to reuse it. But would be worth looking at and we can do it if it does not affect readability much.

Reply:
I cannot use `stream_ui_updates.update_add_subscription_elements` because this function makes changes in the UI of existing streams not in the UI creating new streams. But I liked the second approach and updated the PR.

Reply:
Ok, I was thinking of passing container element to it. But now I think a better approach would be to create a new function like `enable_or_disable_add_subscribers_elements` with `container_elem` (and probably `is_stream_creation` too) as paramters and then call this both from `stream_ui_updates.update_add_subscription_elements` and this.

And this function would only handle enabling and disabling of elements and probably tooltips as well if it does not affect readability.

This change might require some small prep commits to keep it easy to review.